gp-templates/helper-functions.php

```diff
@@ -212,7 +212,7 @@ function textareas( $entry, $permissions, $index = 0 ) {
 
 function display_status( $status ) {
 	$status = preg_replace( '/^[+-]/', '', $status);
-	return $status ? $status : __( 'untranslated', 'glotpress' );
+	return $status ? $status : _x( 'Untranslated', 'Single Status', 'glotpress' );
```

Comment:
All of the status labels are lower case at the moment, I'd leave this one alone.

Reply:
All the status labels are needing a review because are hardcoded.
Still, if kept in lowercase, it's important to have the context.

Reply:
Yep, no issue with the context, the labels are lower case because that's how they're stored in the database and they are case sensitive.  Bad design decision, but a real pain to fix now.

Reply:
Yep, I understand, totally forgot about that.